arch/arc/core/CMakeLists.txt

```diff
@@ -3,28 +3,35 @@
 zephyr_library()
 
 zephyr_library_sources(
-	thread.c
-	thread_entry_wrapper.S
-	cpu_idle.S
-	fatal.c
-	fault.c
-	fault_s.S
-	irq_manage.c
-	timestamp.c
-	isr_wrapper.S
-	regular_irq.S
-	switch.S
-	prep_c.c
-	reset.S
-	vector_table.c
-	)
+  thread.c
```

Comment:
Please do not do this kind of indentation/style changes in a commit like this. This kind of changes add a lot of noise. (In some files the changes are only style..)

Reply:
> (In some files the changes are only style..)
I thought I only changed the indentation/style for the cmake files that I modified, but I guess some ended up identical to the initial version after multiple change iterations.

Acknowledged. I will try to keep things minimal next time.

Reply:
That helps, but note that this is also about having as small a commit as possible when git bisect shows you which commit caused a regression.

Reply:
Agree, but it's sort of minimal in the PR, anyways...